src/main/java/net/dv8tion/jda/internal/utils/InvalidTokenException.java

```diff
@@ -0,0 +1,39 @@
+package net.dv8tion.jda.internal.utils;
+
+/**
+ * This class represents an exception which is thrown when the token provided by the user is invalid
+ * Replaces {@link javax.security.auth.login.LoginException}
```

Comment:
```suggestion
```

I don't see why, since it doesn't matter for the user?

Reply:
It does, because previously they would have to catch that exception or add that to the method signature.

Reply:
I still don't think it's useful personally

Reply:
Previously all our users handled that Exception in some way and now those will be useless, so they should know that we use this exception in place of that so they can refactor their code appropriately.

Reply:
The whole point of a major version bump is to be able to jump without leaving any breadcrumbs behind.

And besides, this would be a reverse breadcrumb, it's not like people using LoginException would know to look here, given the comment is _already_ here.

Reply:
Yes